Gemfile.lock

```diff
@@ -63,7 +63,7 @@ GEM
       faraday (~> 1.0)
     fast_blank (1.0.0)
     fastimage (2.1.7)
-    ffi (1.13.1)
+    ffi (1.12.2)
```

Comment:
Can we apply this lock in the Gemfile, so someone in the future can avoid it?

Reply:
Good idea.